### PR TITLE
Enable show ip bgp on sup and -n all for show ip bgp network

### PR DESCRIPTION
**Pull request by BYGX-wcr**
#### What I did

1. Enable "show ip bgp" on sup and "-n all" for show ip bgp network.
2. Modify README.doc to make the instructions of building and installing the wheel package more clear.
3. Improve the output format of rexec command

#### How I did it

Modify the code in show/main.py to enable "show ip bgp ..." on supervisors and modify show/bgp_frr_v4.py to add support for the new features.
Update README.md.
Modify the rexec implementation to improve the output format.
Add unit tests for the above change.

#### How to verify it

Run on a SONiC chassis

### CODE REVIEW

File: show/bgp_frr_v4.py [hunk 1]
```diff
@@ -102,10 +110,16 @@ def neighbors(ipaddress, info_type, namespace):
...
-    if multi_asic.is_multi_asic() and namespace not in multi_asic.get_namespace_list():
-        ctx = click.get_current_context()
-        ctx.fail('-n/--namespace option required. provide namespace from list {}'\
-            .format(multi_asic.get_namespace_list()))
+    namespace = namespace.strip()
+    if multi_asic.is_multi_asic():
+        if namespace == multi_asic.DEFAULT_NAMESPACE:
+            ctx = click.get_current_context()
+            ctx.fail('-n/--namespace option required. provide namespace from list {}'
+                    .format(multi_asic.get_namespace_list()))
```
Comment (wenyiz2021) [inline show/bgp_frr_v4.py:118]: add 1 more space at beginning of line 118 and 122

File: tests/remote_show_test.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+
+MULTI_LC_REXEC_OUTPUT = '''Since the current device is a chassis supervisor, this command will be executed remotely on all linecards
+hello world
+'''
+
+MULTI_LC_ERR_OUTPUT = '''Since the current device is a chassis supervisor, this command will be executed remotely on all linecards
+Error
+'''
+
```
Comment (wenyiz2021) [inline tests/remote_show_test.py:26]: need 1 more line -- typically 2 empty lines between each class/function

Comment (BYGX-wcr) [reply tests/remote_show_test.py:26]: done

File: show/bgp_frr_v4.py (hunk 1 above)
Comment (BYGX-wcr) [reply show/bgp_frr_v4.py]: done

File: rcli/rexec.py [hunk 3]
```diff
@@ -30,7 +30,7 @@ def cli(linecard_names, command, username):
 
     if list(linecard_names) == ["all"]:
         # Get all linecard names using autocompletion helper
-        linecard_names = rcli_utils.get_all_linecards(None, None, "")
+        linecard_names = sorted(rcli_utils.get_all_linecards(None, None, ""))
```
Comment (wenyiz2021) [inline rcli/rexec.py:33]: please change this linecard_names as well?

Comment (BYGX-wcr) [reply rcli/rexec.py]: done

**Review by wenyiz2021 — APPROVED** [review]

**Comment by wenyiz2021** [comment]
@arlakshm @tjchadaga for viz, there is change in rexec to sort output LC names per request

**Comment by gechiang** [comment]
@BYGX-wcr , I am not able to cleanly cherry-pick this into the msft repo 202205 branch.
Please manually submit a new PR in this REPO:
https://github.com/sonic-net/sonic-utilities.msft
branch 202205
Please tag me to review your new PR.
Thanks!

**Comment by gechiang** [comment]
@BYGX-wcr , Please create a MSFT ADO for this and share its number her so we can track what is being asked to backport to which branch and which one has completed.

**Pull request merged**